app/controllers/create_pdf_file.py

```diff
@@ -0,0 +1,35 @@
+import os
+import io
+
+from flask import render_template
+
+from .price import calculate_subtotal
+
+from app.models import Bid, LinkWorkItem, WorkItemGroup
+import pdfkit
```

Comment:
incorrect import sequence

Reply:
pdfkit it's not standart library, is it shoud be above?
like that:
```
import pdfkit
from .price import calculate_subtotal
from app.models import Bid, LinkWorkItem, WorkItemGroup
```